### DISCUSSION TITLE
Login Issue with https://potres2020.openit.hr/

### DISCUSSION THREAD

**Discussion by kkingkreso** [ushahidi/platform#4215] [General]
Hi,
I have an issue login into https://potres2020.openit.hr/. Password reset is not working.
I would need to make changes to a post but this is not working.

Looking for your support and thanks in advance

Kresi

**Comment by tuxpiper** [comment]
Hi @kkingkreso ! 

For password reset, it would be very important that e-mail delivery is correctly configured in your installation of platform. Is that something that you could check?
